flake.nix

```diff
@@ -16,28 +16,32 @@
       url = "github:nix-community/naersk";
       inputs = {
         nixpkgs.follows = "nixpkgs";
-        flake-utils.follows = "utils";
       };
     };
-
-    mach-nix = {
-      url = "github:DavHau/mach-nix/3.5.0";
-      inputs.nixpkgs.follows = "nixpkgs";
-      inputs.flake-utils.follows = "utils";
-      inputs.pypi-deps-db.follows = "pypi-deps-db";
-    };
-
-    pypi-deps-db = {
-      url = "github:DavHau/mach-nix/3.5.0";
-    };
```

Comment:
use standard Python packaging for nix, since all dependencies (pending `screed`, [PR](https://github.com/NixOS/nixpkgs/pull/205878)) are in `nixpkgs`.